core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -854,6 +874,30 @@ object Parser extends ParserInstances {
   def flatMap01[A, B](pa: Parser[A])(fn: A => Parser1[B]): Parser1[B] =
     Impl.FlatMap1(pa, fn)
 
+  /** Replaces parsed values with the given value.
+    */
+  def as[A, B](pa: Parser[A])(b: B): Parser[B] =
+    map(Impl.Void(pa))(_ => b)
```

Comment:
Actually def void does more than this. It also unmaps pa so we can simplify it. We should use that but I think as I mentioned just using the void method is the way to go.

Reply:
Yeah you are right, it's better to use it.